team-members/iryna-yehorova.html

```diff
@@ -0,0 +1,183 @@
+        It is a small town near Kyiv. Living there is perhaps the best choice I had
+        in my <em>entire</em> life (just for now). <br> I like at the height of 60 m above
+        the ground. Speaking honestlly I hate the height, I am scared of it,
+        but living at the 19th floor is cosy and normal for me. I remember
+        the first months when I was frightened walking near the windows. Now I can even
+        clean them! But only when they are shut &#9786;</p>
+      <br>
+      <h4>What other people think about my favourite town</h4>
+      <p>My mom:</p>
+      <blockquote>
+        Such a good place! I like living near Kyiv. It is like you can go
+        shopping to the big malls and visit the best hospitals. But breeze the
+        fresh air of small town.
+      </blockquote>
+      <br>
+      <p>My husband:</p>
+      <blockquote>
+        I do not like Brovary because I work in Kyiv. Every morning I need to
+        drive to other side of Kyiv to get to work. It is annoying. I prefer to
+        live near my work.
+      </blockquote>
+      <br>
+      <p>My daughter:</p>
+      <blockquote>
+        Blablablabla bla blablabla. Ooh. Pff. Bla bla...
+      </blockquote>
+      <br>
+    </article>
+    <section>
+      <h3 id="contact">Contact me</h3>
+      <form action="#" method="get">
+        Write your name*:
+        <input type="text" name="user_name" placeholder="Name" required>
+        <br>
+        Write yout email:
+        <input type="email" name="user_email" placeholder="Email">
+        <br>
+        Write your telephone*:
+        <input type="tel" name="telephone" placeholder="(088) 888 88 88" pattern="([0-9]{3}-[0-9]{3}-[0-9]{2}-[0-9]{2}"
+          required>
+        <br>
+        Your gender:
+        <input type="radio" name='gender' value="male">Male
+        <input type="radio" name="gender" value="female">Female
+        <input type="radio" name="gender" value="other">Other
+        <br>
+        Your date of birth:
+        <input type="date" name="date_of_birth">
+        <br>
+        Write your opinions, ideas, etc:
+        <textarea width="120"></textarea>
+        <br>
+        <p>with * marked all necessary elements</p>
+        <input type="submit" value="Submit">
+        <input type="reset" value="Reset">
+      </form>
+      <br>
+    </section>
+  </main>
+  <footer>
+    <time datetime="2019-04-09 20:50">04/09/2019</time>
+    <p>&copy; All rights are reserved</p>
+    <p>Me at social networks</p>
+    <ul>
+      <li><a href="https://www.facebook.com/profile.php?id=100017643998366" target="_blank">
+          <img src="../images/facebook.png" alt="Facebook profile" width="40px">
+        </a></li>
+      <li><a href="https://github.com/NervoYe" target="_blank"><img src="../images/Github.png" alt="Github profile"
+            width="50px"></a>
+      </li>
+      <li><a href="https://www.instagram.com/yehorova.irene/?hl=uk" target="_blank"><img src="../images/Instagram.png"
```

Comment:
very long line (review similar cases)
could be:
<li>
  <a href="https://www.instagram.com/yehorova.irene/?hl=uk" target="_blank">
    <img src="../images/Instagram.png" alt="Instagram profile" width="40px">
  </a>
</li>